Fixed some of the cmake buildsystem issues. They still forget to set the
proper version and still seam to not check the Linux builds. CMake
provides an ALSA module for a long time now ...
---
diff -urpN nana.orig/CMakeLists.txt nana/CMakeLists.txt
--- nana.orig/CMakeLists.txt	2019-10-05 08:13:02.018317259 +0200
+++ nana/CMakeLists.txt	2019-10-05 08:13:38.796748368 +0200
@@ -21,11 +21,11 @@
 # https://cmake.org/cmake/help/v3.12/module/CMakeDependentOption.html?highlight=cmakedependentoption
 # cmake 3.12 have more better modern c++ support
 
-cmake_minimum_required(VERSION 3.12  FATAL_ERROR)
-project(nana VERSION      1.7.1
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+project(nana VERSION 1.7.2
         DESCRIPTION "C++ GUI library"
         HOMEPAGE_URL http://nanapro.org
-        LANGUAGES CXX         )
+        LANGUAGES CXX)
 
 #######################     Main setting of Nana targets, sources and installs    #####################
 
diff -urpN nana.orig/build/cmake/enable_audio.cmake nana/build/cmake/enable_audio.cmake
--- nana.orig/build/cmake/enable_audio.cmake	2019-10-05 08:13:02.018317259 +0200
+++ nana/build/cmake/enable_audio.cmake	2019-10-05 08:14:23.775572475 +0200
@@ -4,12 +4,12 @@ option(NANA_CMAKE_ENABLE_AUDIO "Enable c
 if(NANA_CMAKE_ENABLE_AUDIO)
     target_compile_definitions(nana PUBLIC NANA_ENABLE_AUDIO)
     if(UNIX)
-        find_package(ASOUND) # ? https://github.com/hintjens/demidi/blob/master/Findasound.cmake
-        if(ASOUND_FOUND)
-            target_include_directories(nana PUBLIC ${ASOUND_INCLUDE_DIRS})
-            target_link_libraries(nana PUBLIC  ${ASOUND_LIBRARIES})
+        find_package(ALSA) # ? https://github.com/hintjens/demidi/blob/master/Findasound.cmake
+        if(ALSA_FOUND)
+            target_include_directories(nana PUBLIC ${ALSA_INCLUDE_DIRS})
+            target_link_libraries(nana PUBLIC ${ALSA_LIBRARIES})
         else()
             message(FATAL_ERROR "libasound is not found")
         endif()
     endif()
-endif()
\ No newline at end of file
+endif()
